crates/frontend/src/pages/variables_list.rs

```diff
@@ -0,0 +1,151 @@
+        pagination_params_rws.update(|f| f.page = Some(page));
+    });
+
+    let table_columns = create_memo(move |_| {
+        let expand = move |key_name: &str, _row: &Map<String, Value>| {
+            let label = key_name.to_string();
+
+            view! {
+                <A href=label.clone() class="ml-[22px] text-blue-500 underline underline-offset-2">
+                    {label}
+                </A>
+            }
+            .into_view()
+        };
+
+        vec![
+            Column::new(
+                "key".to_string(),
+                false,
+                expand,
+                ColumnSortable::No,
+                Expandable::Disabled,
+                default_column_formatter,
+            ),
+            Column::default("value".to_string()),
+            Column::default("created_at".to_string()),
+            Column::default_with_column_formatter("last_modified_at".to_string(), |_| {
+                default_column_formatter("Modified At")
+            }),
+        ]
+    });
+
+    view! {
+        <Suspense fallback=move || {
+            view! { <Skeleton /> }
+        }>
+            {move || {
+                let variables = variables_resource.get().unwrap_or_default();
+                let table_rows = variables
+                    .data
+                    .into_iter()
+                    .map(|var| {
+                        let mut ele_map = json!(var).as_object().unwrap().to_owned();
+                        ele_map
+                            .insert(
+                                "created_at".to_string(),
+                                Value::String(var.created_at.format("%v %T").to_string()),
+                            );
+                        ele_map
+                            .insert(
+                                "last_modified_at".to_string(),
+                                Value::String(var.last_modified_at.format("%v %T").to_string()),
+                            );
+                        ele_map
+                    })
+                    .collect::<Vec<Map<String, Value>>>();
+                let total_variables = variables.total_items.to_string();
+                let pagination_params = pagination_params_rws.get();
+                let current_page = pagination_params.page.unwrap_or_default();
+                let total_pages = variables.total_pages;
+                let pagination_props = TablePaginationProps {
+                    enabled: true,
+                    count: pagination_params.count.unwrap_or_default(),
+                    current_page,
+                    total_pages,
+                    on_page_change: handle_page_change,
+                };
+
+                view! {
+                    <div class="h-full flex flex-col gap-4">
+                        <div class="flex justify-between">
+                            <Stat
+                                heading="Variables"
+                                icon="ri-code-box-line"
```

Comment:
can we rather use this
code box is in use for functions
```suggestion
                                icon="ri-braces-line"
```